PROTOCOL.md

```
@@ -24,7 +25,7 @@ Historical drafts and analysis notes are listed in [docs/NON_CANONICAL.md](docs/
 |--------|------|--------|
 | 0 | 4 | magic |
 | 4 | 4 | version (`0x01`) |
-| 8 | 32 | parent |
+| 8 | 32 | parent | Previous `Message.id` or genesis zeros |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)